ts/features/payments/home/screens/PaymentsHomeScreen.tsx

```diff
@@ -1,3 +1,71 @@
+import { HeaderFirstLevel, VSpacer } from "@pagopa/io-app-design-system";
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { useFocusEffect } from "@react-navigation/native";
 import * as React from "react";
+import { createSelector } from "reselect";
+import I18n from "../../../../i18n";
+import { useIODispatch, useIOSelector } from "../../../../store/hooks";
+import { walletPaymentGetUserWallets } from "../../payment/store/actions/networking";
+import { walletPaymentUserWalletsSelector } from "../../payment/store/selectors";
+import PaymentHistorySection from "../components/PaymentsHomeScreenHistorySection";
+import PaymentMethodsSection from "../components/PaymentsHomeScreenMethodsSection";
+import { walletTransactionHistorySelector } from "../store/selectors";
+import { useTransactionHistory } from "../utils/transactionHistoryHandler";
 
-export const PaymentsHomeScreen = () => <></>;
+/* "rendering truth table" for both pots goes as such
+ *          SOME | NONE
+ * LOADING    X  |  X
+ * ERROR      X  |  O
+ * -          X  |  O
+ *
+ * we need to make sure that only in O case we render the empty state
+ */
+const anySectionSomeOrLoading = createSelector(
+  walletPaymentUserWalletsSelector,
+  walletTransactionHistorySelector,
+  (userWallets, transactions) => {
+    const shouldRenderMethods =
+      pot.isSome(userWallets) || pot.isLoading(userWallets);
+    const shouldRenderHistory =
+      pot.isSome(transactions) || pot.isLoading(transactions);
+    return shouldRenderMethods || shouldRenderHistory;
+  }
+);
```

Comment:
Why did you place a selector here and not in the `selectors.ts` file?

Reply:
since it is an extremely specific selector, I think it's better to just keep it in the screen's scope instead of cluttering the selectors file; what do you think?